src/lib/stitching/causality/stitching.ts

```diff
@@ -118,16 +118,47 @@ export const causalityStitchingEnvironment = ({
           // (eg first, after, last, before) are forwarded automatically, so we only
```

Comment:
Is that true? I would expect this to need a 

```
args: {
  userID: parent.internalID,
  ..._args
}
```

to make sure those connection args get passed through.

Reply:
I didn't verify, but from the [doc](https://www.graphql-tools.com/docs/schema-delegation/#args-recordstring-any), the arguments are automatically preserved if the subschema is expecting them.

Reply:
passing `first` to connection works so I _think_ they are implicitly being passed.